.npmignore

```diff
@@ -0,0 +1 @@
+test/
```

Comment:
It's fine but a new line would be better

Reply:
Added!